src/widgets/widget.ts

```diff
@@ -60,6 +65,10 @@ export class Widget {
 
 
     // Search list of elements
+    public allBySystelabTestId(dataTestId: string): ElementArrayFinder {
+        return this.elem.allByCSS(`[${Test.SYSTELAB_TEST_ID_ATTRIBUTE}="${dataTestId}"]`);
```

Comment:
What about calling to the new method added at ElementFinder level > allBySystelabTestId()

Reply:
Applied